packages/broker/test/unit/plugins/operator/closeExpiredFlags.test.ts

```diff
@@ -0,0 +1,41 @@
+import { closeExpiredFlags } from '../../../../src/plugins/operator/closeExpiredFlags'
+import { ContractFacade, SponsorshipResult } from '../../../../src/plugins/operator/ContractFacade'
+import { mock } from 'jest-mock-extended'
+import { toEthereumAddress } from '@streamr/utils'
+
+const testAddress = toEthereumAddress('0xb85ea99a770d3d79d03855045cad830ef028a024') // some random address
```

Comment:
I would differentiate between sponsorship address and operator address to make test assertion a bit stronger.

```
const sponsorshipAddress = toEthereumAddress('0xb85ea99a770d3d79d03855045cad830ef028a024')
const operatorAddress = toEthereumAddress('0x9b9dA31a92fA066A193fc78aE55Ee3eafa865aC7')
```